tests/snuba/api/endpoints/test_organization_events.py

```diff
@@ -4993,3 +4993,12 @@ def test_no_ratelimit(self):
                 self.do_request(query)
             response = self.do_request(query)
             assert response.status_code == 200, response.content
+
+    def test_transaction_source(self):
+        query = {
+            "field": ["transaction"],
+            "query": "transaction.source:task",
+            "project": [self.project.id],
+        }
+        response = self.do_request(query)
+        assert response.status_code == 200, response.content
```

Comment:
we just need to wait for new snuba version.

Reply:
this test rely on this snuba commit https://github.com/getsentry/snuba/commit/9e7b0eca29b76778f7168ef4777af836f3ac2618